runtime-modules/content/src/lib.rs

```diff
@@ -90,6 +91,25 @@ use sp_std::{borrow::ToOwned, collections::btree_set::BTreeSet, vec::Vec};
 
 type WeightInfoContent<T> = <T as Config>::WeightInfo;
 
+/// The log target of this pallet.
+pub const LOG_TARGET: &str = "runtime::content";
+
+// syntactic sugar for logging.
+#[macro_export]
+macro_rules! log {
```

Comment:
I will try, although I think the macro export statement might only work in the crate root.
Although we don't use the log statement anywhere else, it could be useful if we want to add some logs in future?